tests/ui/issues/issue-39175.rs

```diff
@@ -4,8 +4,6 @@
 // these platforms also.
 
 //@ ignore-windows
-//@ ignore-wasm32 no processes
-//@ ignore-sgx no processes
 
 use std::process::Command;
```

Comment:
This test is actually exercising closure type inference, it does not need to run at all